README.rst

```diff
@@ -149,8 +390,51 @@ To remedy the situation, change the data stored in the keyring as follows:
 You may be prompted for your user login password by your operating
 system, depending on how your key store is configured.
 
-**Windows issues**
+Command line flag ``--ecp-endpoint-url`` error parsing parameter
+----------------------------------------------------------------
+
+If you encounter the following error it is because the awscli expects
+urls passed as arguments to return a 200 on an HTTP GET (See
+`aws-cli#4473 <https://github.com/aws/aws-cli/issues/4473>`_)::
+
+    $ aws login --ecp-endpoint-url https://shibboleth.illinois.edu/idp/profile/SAML2/SOAP/ECP
+    Error parsing parameter '--ecp-endpoint-url': Unable to retrieve https://shibboleth.illinois.edu/idp/profile/SAML2/SOAP/ECP: received non 200 status code of 500
+
+This check can be disabled on a per profile basis using the following
+command::
+
+    $ aws configure set cli_follow_urlparam off
+
+Windows issues
+--------------
+
+Auto-renewal is not supported under the Windows PowerShell, Command
+Prompt, or Git Shell for Windows. Auto-renewal is supported under
+the Windows Subsystem for Linux (WSL).
+
+GitBash bad interpreter errors
+``````````````````````````````
+
+If you receive a bad interpreter error from the aws command it may
+be because you have a space in the path of your Python interpreter::
+
+    bash: /c/Users/me/AppData/Roaming/Python/Python38/Scripts/aws: c:\program: bad interpreter: No such file or directory
+
+To fix this issue either reinstall your Python interpreter to a
+path that does not contain a space and then reinstall the awscli
+package, or more simply just define an alias in your bashrc file::
+
+    alias aws='python $(which aws)'
+
+Windows Subsystem for Linux bad interpreter error
+`````````````````````````````````````````````````
+
+If you receive a bad interpreter error from the WSL it may be because
+the location where the awscli is installed is not listed before the
+location of the Windows install of awscli::
+
+    -bash: /mnt/c/Python39/Scripts/aws: c:\python39\python.exe^M: bad interpreter: No such file or directory
 
-Windows PowerShell, Command Prompt, and Git Shell for Windows are not
-currently supported because of problems with auto-renewal of AWS credentials,
-and other known issues.
+To remedy this issue please ensure that the location where the
+awscli is installed in the WSL comes before the location of the
+Windows install.
```

Comment:
Should we clarify here that WSL location needs to come before the Windows location in the PATH environment variable?
```suggestion
To remedy this issue please ensure that the location where the
awscli is installed in the WSL comes before the location of the
Windows install in the PATH environment variable.
```